cmd/fetch.go

```diff
@@ -24,6 +26,10 @@ import (
 	"github.com/spf13/pflag"
 )
 
+const (
+	localAgentStateURL = "http://127.0.0.1:5051/state"
```

Comment:
double check if people can change port 5051. If not, can we do better?

Reply:
Defaults are meant for user friendliness. Option to change port/url makes sense for a command.

Reply:
If you can, please create the option for this.
Otherwise, you can put TODO there for futurework.

Reply:
Since this is not a fetching from agent command, I don't think the option to change port makes sense here. If fetching from agent is added as a separate command (other than Zookeeper), then we can add that option.